pkgs/development/compilers/abcl/default.nix

```diff
@@ -1,38 +1,67 @@
-{lib, stdenv, fetchurl, ant, jre, jdk}:
+{ stdenv
+, lib
+, fetchurl
+, ant
+, jre
+, jdk
+, makeWrapper
+}:
+
 stdenv.mkDerivation rec {
   pname = "abcl";
   version = "1.9.1";
-  # or fetchFromGitHub(owner,repo,rev) or fetchgit(rev)
+
   src = fetchurl {
     url = "https://common-lisp.net/project/armedbear/releases/${version}/${pname}-src-${version}.tar.gz";
     sha256 = "sha256-pbxnfJRB9KgzwgpUG93Rb/+SZIRmkd6aHa9mmfj/EeI=";
   };
+
   configurePhase = ''
+    runHook preConfigure
+
     mkdir nix-tools
     export PATH="$PWD/nix-tools:$PATH"
     echo "echo nix-builder.localdomain" > nix-tools/hostname
     chmod a+x nix-tools/*
 
     hostname
+
+    runHook postConfigure
   '';
+
+  buildInputs = [ jre ant jdk jre ];
```

Comment:
Even though you did just move it, maybe this is a good opportunity to move `ant` and `jdk` into `nativeBuildInputs` because I dont think abcl needs those at runtime and only at build time.

Reply:
I think it's too late for this PR

That can be solved in a follow up PR

Reply:
I'm working on it though